airgun/widgets.py

```diff
@@ -676,6 +676,7 @@ def select_org(self, org_name):
         )
         self.browser.plugin.ensure_page_safe()
 
+    @property
```

Comment:
should not you change https://github.com/SatelliteQE/airgun/blob/master/airgun/entities/organization.py#L87 too?

Reply:
No idea how pycharm's search didn't reveal that to me. Big thanks for catching!